PhysicsTools/PatAlgos/python/slimming/hiPixelTracks_cfi.py

```diff
@@ -3,6 +3,11 @@
 from PhysicsTools.PatAlgos.patTracksToPackedCandidates_cfi import patTracksToPackedCandidates
 
 hiPixelTracks = patTracksToPackedCandidates.clone()
+hiPixelTracks.dxySigCut = 3.
```

Comment:
please update parameters during cloning
```suggestion
hiPixelTracks = patTracksToPackedCandidates.clone(
  dxySigCut = 3.
)
```
(similarly for the other parameters)

Reply:
Thanks @slava77 , I fixed this doing this commit https://github.com/cms-sw/cmssw/pull/36558/commits/f3d9ca78f2748d6108378188f1a7b6c80ed2d5dd